old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/04-intro-CRS-projection.Rmd

```diff
@@ -116,35 +117,32 @@ To define the location of something we often use a coordinate system. This syste
 consists of an X and a Y value, located within a 2 (or more) -dimensional space.
 
 <figure>
-	<a href="http://open.senecac.on.ca/clea/label/projectImages/15_276_xy-grid.jpg">
-	<img src="http://open.senecac.on.ca/clea/label/projectImages/15_276_xy-grid.jpg"></a>
+	<a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/vector-general/2D_Coordinates.jpg">
+	<img src="https://openpress.usask.ca/introgeomatics/chapter/frames-of-reference-and-coordinate-systems/" alt="An X, Y grid with points expressed as ordered pairs."></a>
 	<figcaption> We use coordinate systems with X, Y (and sometimes Z axes) to
 	define the location of objects in space.
-	Source: http://open.senecac.on.ca
+	Source: https://openpress.usask.ca/
 	</figcaption>
 </figure>
 
-While the above coordinate system is 2-dimensional, we live on a 3-dimensional
-earth that happens to be "round". To define the location of objects on the earth, which is round, we need
-a coordinate system that adapts to the Earth's shape. When we make maps on paper
-or on a flat computer screen, we move from a 3-Dimensional space (the globe) to
-a 2-Dimensional space (our computer
-screens or a piece of paper). The components of the CRS define how the
-"flattening" of data that exists in a 3-D globe space. The CRS also defines the
-the coordinate system itself.
+While the above coordinate system is 2-dimensional, we live on a 3-dimensional 
+earth that happens to be "round". To define the location of objects on the earth, 
+which is round, we need a coordinate system that adapts to the Earth's shape. When 
+we make maps on paper or on a flat computer screen, we move from a 3-Dimensional 
+space (the globe) to a 2-Dimensional space (our computer screens or a piece of paper). 
+The components of the CRS define how the "flattening" of data that exists in a 3-D 
+globe space. The CRS also defines the the coordinate system itself.
 
 <figure>
 	<a href="http://ayresriverblog.com/wp-content/uploads/2011/05/image.png">
-	<img src="http://ayresriverblog.com/wp-content/uploads/2011/05/image.png"></a>
+	<img src="http://ayresriverblog.com/wp-content/uploads/2011/05/image.png"
+	alt="A point on the globe on the left translates to an X and Y point on a flat map on the right."></a>
 	<figcaption>A CRS defines the translation between a location on the round earth
 	and that same location, on a flattened, 2 dimensional coordinate system.
 	Source: http://ayresriverblog.com
 	</figcaption>
 </figure>
 
-> A coordinate reference system (CRS) is a
-coordinate-based local, regional or global system used to locate geographical
-entities. -- Wikipedia
```

Comment:
@donal-at-NEON: this didn't seem necessary and I think other text explains what a CRS is just fine. Just want to make sure this text isn't absolutely needed.